apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/open_trades/trade_details/TradeDetailsView.java

```diff
@@ -0,0 +1,223 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.desktop.main.content.bisq_easy.open_trades.trade_details;
+
+import bisq.desktop.common.Layout;
+import bisq.desktop.common.view.NavigationView;
+import bisq.desktop.components.controls.BisqIconButton;
+import bisq.desktop.components.controls.MaterialTextField;
+import bisq.desktop.overlay.OverlayModel;
+import bisq.i18n.Res;
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
+import javafx.geometry.Insets;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextArea;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Priority;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+
+import lombok.extern.slf4j.Slf4j;
+import javafx.scene.control.ScrollPane;
+
+@Slf4j
+public class TradeDetailsView extends NavigationView<AnchorPane, TradeDetailsModel, TradeDetailsController> {
+
+    private final Button closeButton;
+    private final ScrollPane scrollPane;
+
+    private final VBox allInfoContainer, informationContainer, detailsContainer;
+
+    private final DetailsTextRow myUsername, peerUsername, tradeId, myRole;
+    private final DetailsTextRow tradeAmountFiat, fiatPaymentMethod;
+    private final DetailsTextRow bitcoinPaymentMethod, marketPrice, offerTakenTime;
+    private final DetailsTextRow myNetworkAddress, peerNetworkAddress;
+    private final DetailsRow tradeAmountBTC, paymentAccountData, bitcoinPaymentAddress;
+
+    private static final int ROW_HEIGHT = 40;
+
+    public TradeDetailsView(TradeDetailsModel model, TradeDetailsController controller) {
+        super(new AnchorPane(), model, controller);
+
+        root.setPrefWidth(OverlayModel.WIDTH);
+        root.setPrefHeight(OverlayModel.HEIGHT + 200);
+        int rowSpacing = 2;
+
+        allInfoContainer = new VBox(25);
+        scrollPane = new ScrollPane(allInfoContainer);
+        scrollPane.setFitToWidth(true);
+        scrollPane.setPadding(new Insets(15, 15, 15, 15));
+        allInfoContainer.setPadding(new Insets(30, 30, 30, 30));
+        allInfoContainer.prefHeightProperty().bind(root.heightProperty());
```

Comment:
bindings must be set in onViewAttached. Also follow established style to use onViewAttached and onViewDetached after constructor as those are the most important methods.

All bindings, event listeners, event handlers must be deactivated in onViewDetached. See the dev docs as well or other code.